src/Tablerow.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState } from "react";
+import Moment from "moment";
+
+function CreateTableRow({ profile, index, showProfile }) {
```

Comment:
creating a new component for each table row works really well, this means you can easily control the state of each row when deciding if the row should be highlighted or not